lab_1_keywords_tfidf/main.py

```diff
@@ -66,7 +93,20 @@ def get_top_n(frequencies: dict[str, Union[int, float]], top: int) -> Optional[l
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not isinstance(frequencies, dict) or not isinstance(top, int) or isinstance(top, bool) or frequencies == {}:
+        return None
+    if top <= 0:
+        return None
+    for i in frequencies.items():
+        if not isinstance(i[0], str) or not isinstance(i[1], (int, float)):
+            return None
+    sorted_list = []
+    sort = sorted(frequencies.items(), reverse=True, key=lambda x: x[1])
+    for i in sort:
```

Comment:
Why do you do this?

Reply:
if you're talking about line 104, i wanted to get a list of values from frequencies in a descending order